README.md

```diff
@@ -259,14 +259,24 @@ In other scenarios, a component may have an initial loading state (e.g. loading
 
 ```javascript
 class SlowElem extends LitElement {
-  render() {
-    return html`<p>I take my time</p>`;
+  constructor() {
+    super();
+    this._loadingCompleteResolve = undefined;
```

Comment:
This shouldn't be necessary, right?

Reply:
Yeah it's not -- I've removed it. I normally like including it just to be explicit about what private members a class has (even if they're initially `undefined`), but it does clutter the example and is technically redundant.